nix/build/ocaml-overlay.nix

```
@@ -10,19 +10,19 @@ in
 with opam-nix.lib.${self.system}; let
   zcash-overlay = import ./zcash-overlay.nix;
   hacks = import ./hacks.nix;
-  tezosSourcesResolved =
-    self.runCommand "resolve-tezos-sources" {} "cp --no-preserve=all -Lr ${sources.tezos} $out";
-  tezosScope = buildOpamProject' {
+  octezSourcesResolved =
+    self.runCommand "resolve-octez-sources" {} "cp --no-preserve=all -Lr ${sources.tezos} $out";
+  octezScope = buildOpamProject' {
     repos = [sources.opam-repository];
     recursive = true;
     resolveArgs = { };
-  } tezosSourcesResolved { };
+  } octezSourcesResolved { };
 in {
-  tezosPackages = (tezosScope.overrideScope' (pkgs.lib.composeManyExtensions [
+  octezPackages = (octezScope.overrideScope' (pkgs.lib.composeManyExtensions [
       (_: # Nullify all the ignored protocols so that we don't build them
         builtins.mapAttrs (name: pkg:
           if builtins.any
-          (proto: !isNull (builtins.match "tezos.*${proto}.*" name))
+          (proto: !isNull (builtins.match ".*${proto}.*" name))
```

this looks a bit odd, shouldn't it be:
```suggestion
          (proto: !isNull (builtins.match "octez.*${proto}.*" name))
```

---

why we need `tezos` here if we just looking up for protocol shorthand? also `tezos-admin-client` wasn't renamed (but it is not relevant here however)

---

fair enough :ok_hand: